src/include/distributed/master_metadata_utility.h

```diff
@@ -25,6 +25,9 @@
 /* total number of hash tokens (2^32) */
 #define HASH_TOKEN_COUNT INT64CONST(4294967296)
 #define SELECT_EXIST_QUERY "SELECT EXISTS (SELECT 1 FROM %s)"
+#define INNER_SHARD_TABLE_SIZE_QUERY "pg_table_size(%s)"
```

Comment:
We have SHARD_TABLE_SIZE_QUERY (which is exactly a pg_table_size query) constant defined in master_protocol.h and used to implement WorkerShardStats. I have named these queries like that since there are not sent directly to workers as a query, but as a part of SELECT query.